docs/developer-docs/latest/development/plugins/graphql.md

```diff
@@ -255,6 +247,7 @@ You can also apply different parameters to the query to make more complex querie
 - `start` (integer): Define the amount of entries to skip.
 - `sort` (string): Define how the data should be sorted.
 - `publicationState` (PublicationState): Only select entries matching the publication state provided.
+- `locale` (string): Define the locale to fetch the content for, if the [Internationalization (i18n) plugin](/developer-docs/latest/development/plugins/i18n.md) is installed
```

Comment:
same or maybe focus on i18n begin enabled on the Content type ?

Reply:
Thanks for the review, @alexandrebodin ! 
I've just added both suggestions in [this commit](https://github.com/strapi/documentation/pull/224/commits/6f9d37266dd5ecb80b1320cf6ab06fd17a4e4ccd)